demo/spec/home.spec.js

```diff
@@ -1,8 +1,14 @@
+/* eslint-env jest */
+
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { Home } from '../src/Home';
+import ConnectedHome from '../src/home';
+
+const { WrappedComponent: Home } = ConnectedHome;
```

Comment:
Thanks! I thought so, too - doing so doesn't increase the exposed API surface...